yt_dlp/extractor/twitter.py

```diff
@@ -1165,11 +1148,11 @@ def _build_graphql_query(self, space_id):
 
     def _real_extract(self, url):
         space_id = self._match_id(url)
-        space_data = self._call_graphql_api('HPEisOmj1epUNLCWTYhUWw/AudioSpaceById', space_id)
+        space_data = self._call_graphql_api('HPEisOmj1epUNLCWTYhUWw/AudioSpaceById', space_id)['audioSpace']
         if not space_data:
             raise ExtractorError('Twitter Space not found', expected=True)
 
-        metadata = space_data['audioSpace']['metadata']
+        metadata = space_data['metadata']
         live_status = try_call(lambda: self.SPACE_STATUS[metadata['state'].lower()])
```

Comment:
I think I like `traverse_obj` more for this.
```suggestion
        live_status = traverse_obj(self.SPACE_STATUS, metadata.get('state'), casesense=False)
```